src/views/facilities/dashBoardContainer.js

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import SideBar from './containers/sideBar';
+import Appbar from './containers/appBar'
+import { makeStyles } from '@material-ui/core/styles';
+import Facility from './facility'
+
+
+const useStyles = makeStyles((theme) => ({
+  _wrapper:{
+      marginLeft:'210px',
+      marginTop:'64px',
+      backgroundColor:'#fff!important',
+      padding:'40px',
+      height:'100vh' 
+  },
+  _sideBar:{
+    '@media (min-width:780px)': {
+      display:'none'
+    }
+  }
+ 
+  
+}));
+
+const DashboardContainer=()=>{
+  const classes = useStyles();
+  return(
+    <div>
+      <Appbar/>
+      <div className={classes._wrapper}>
+        <Facility />
+      </div>
+      <SideBar className={classes._sideBar} />
+    </div>
```

Comment:
Since the sideBar will not be Persistent by default could you just make the table fit the available space just as to avoid overflowX at large screen size